packages/config/scripts/checkVerifiedContracts/check.ts

```diff
@@ -0,0 +1,52 @@
+import { Logger } from '@l2beat/backend-tools'
+
+import { bridges, layer2s } from '../../src'
+import { getManuallyVerifiedContracts } from '../../src/verification/manuallyVerifiedContracts'
+import {
+  areAllProjectContractsVerified,
+  getUniqueContractsForAllProjects,
+} from './addresses'
+import { getEtherscanClient } from './etherscan'
+import {
+  getOutputPath,
+  loadPreviouslyVerifiedContracts,
+  PROJECTS_OUTPUT_PATH,
+  saveResult,
+  VerificationMap,
+  VerificationMapPerChain,
+} from './output'
+import { verifyContracts } from './tasks'
+
+export const SUPPORTED_CHAINS = ['ethereum', 'arbitrum']
```

Comment:
I'd extract that to some configuration file higher up, it's kinda weird that this is here